programs/fileio.h

```diff
@@ -96,22 +103,27 @@ void FIO_setNotificationLevel(int level);
 void FIO_setExcludeCompressedFile(FIO_prefs_t* const prefs, int excludeCompressedFiles);
 void FIO_setPatchFromMode(FIO_prefs_t* const prefs, int value);
 void FIO_setContentSize(FIO_prefs_t* const prefs, int value);
-void FIO_setNbFiles(FIO_prefs_t* const prefs, int value);
-void FIO_setCurrFileIdx(FIO_prefs_t* const prefs, int value);
+
+/* FIO_ctx_t functions */
+void FIO_setNbFilesTotal(FIO_ctx_t* const fCtx, int value);
```

Comment:
So, I'm aware that I never actually use these functions - is it more idiomatic to have `set()`-like functions for structs or to just not have them in the first place?

Reply:
We don't need them. They are present for `FIO_prefs_t` so that callers can set fields in `FIO_prefs_t` without having the struct definition.

We only need these functions if someone outside of `fileio.c` needs to set a field on the `FIO_ctx_t`.

Reply:
Ah that makes sense - I was also wondering why we always define the struct with a `_s` suffix and typedef it into `_t`, but that seems to explain that as well!